packages/wonder-blocks-button/components/button.md

```diff
@@ -83,181 +131,325 @@ const styles = StyleSheet.create({
+    >
+        Secondary
+    </Button>
     <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="tertiary"
-    >Label</Button>
+    >
+        Tertiary
+    </Button>
     <Button
-        disabled={true}
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
-    >Disabled</Button>
-    <Button
         disabled={true}
+    >
+        Primary
+    </Button>
+    <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="secondary"
-    >Disabled</Button>
-    <Button
         disabled={true}
+    >
+        Secondary
+    </Button>
+    <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="tertiary"
-    >Disabled</Button>
+        disabled={true}
+    >
+        Tertiary
+    </Button>
 </View>
 ```
 
-Button examples, `style` specified (Supports width, position, margin, and flex styles):
+Buttons have a `size` that's either `"medium"` (default) or `"small"`.
 ```js
 const {View} = require("@khanacademy/wonder-blocks-core");
 const {StyleSheet} = require("aphrodite");
 
 const styles = StyleSheet.create({
     row: {
-        display: "row",
+        flexDirection: "row",
     },
-    wideButton: {
-        width: 200,
+    button: {
+        marginRight: 10,
+    }
+});
+
+<View style={styles.row}>
+    <Button
+        style={styles.button}
+        onClick={(e) => console.log("Hello, world!")}
+        size="small"
+    >
+        Label
+    </Button>
+    <Button
+        style={styles.button}
+        onClick={(e) => console.log("Hello, world!")}
+        kind="secondary"
+        size="small"
+    >
+        Label
+    </Button>
+    <Button
+        style={styles.button}
+        onClick={(e) => console.log("Hello, world!")}
+        kind="tertiary"
+        size="small"
+    >
+        Label
+    </Button>
+</View>
+```
+
+Buttons can have an `href` or an `onClick` handler or both.
+
+Being able to use both is necessary to support marking conversions in A/B
+tests.  There is however no built-in facility for doing A/B testing in
+`wonder-blocks` itself.
+```jsx
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
     },
-    centerInDiv: {
-        display: 'block',
-        margin: '0 auto',
+    button: {
+        marginRight: 10,
+    }
+});
+
+<View style={styles.row}>
+    <Button
+        href="#button-1"
+        style={styles.button}
+    >
+        href
+    </Button>
+    <Button
+        kind="secondary"
+        onClick={(e) => console.log("Hello, world!")}
```

Comment:
When I clicked these I was kind of surprised when something didn't happen. Maybe an `alert` might be more obvious?